### PR TITLE
T1 add

### PR DESCRIPTION
**Pull request by IhorYutinDev**
add hello

### CODE REVIEW

File: src/main/java/com/javarush/jira/ihor/New.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.javarush.jira.ihor;
+
+public class New {
```
Comment (foretruff) [inline src/main/java/com/javarush/jira/ihor/New.java]: Bad naming , refactor

**Review by foretruff — COMMENTED** [review]
Fix comments

File: src/main/java/com/javarush/jira/ihor/New.java (hunk 1 above)
Comment (IhorYutinDev) [reply src/main/java/com/javarush/jira/ihor/New.java]: I've fixed

**Review by foretruff — APPROVED** [review]
Aproved

**Pull request merged**